spec/views/hq/sponsors/index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe "hq/sponsors/index.html.haml", type: :view do
+  let(:sponsors) do
+    [ FactoryGirl.build_stubbed(:sponsor), FactoryGirl.build_stubbed(:sponsor)]
+  end
+
+  describe 'sponsors exist' do
+    before :each do
+      assign(:sponsors, sponsors)
```

Comment:
In view specs, since instance variables are passed to the view, there seems to be no advantage to breaking up the setup into `let` & `before` over
```ruby
before(:each) do
  @sponsors = [FactoryGirl.build_stubbed(:sponsor)]
end
```